client/src/Header/header.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import playLogo from './Images/play_logo.png';
+import './header.style.css';
+
+class Header extends Component {
+    render() {
+        return (
+            <div className="header">
+                <ul className="nav-bar">
+                    <li className="nav-element" id="logo">
```

Comment:
let's avoid ids unless needed by javascript. it's good practice to use class names only for styling